test/parallel/test-http-parser.js

```diff
@@ -129,7 +129,7 @@ function expectBody(expected) {
 //
 {
   const request = Buffer.from(
-      'HTTP/1.1 200 OK' + CRLF +
+    'HTTP/1.1 200 OK' + CRLF +
```

Comment:
Will square them all up. Might also take the opportunity to get rid of the `CRLF` constant...